group_vars/all.yml

```diff
@@ -123,22 +123,24 @@ dependencies_version:
   postgresql_connector: 42.2.19
   postgres_major_version: 14
   activemq: 5.16.4
-  java: 11.0.15
-  java_build: 10
+  java: 17.0.3
+  java_build: 7
   tomcat: 9.0.59
   libreoffice: 7.2.5.1
   pdf_renderer: 1.1
   imagemagick: 7.1.0-16
 apache_archive_url: https://archive.apache.org
+java_major: "{{ dependencies_version.java | regex_search('^[0-9]+') }}"
```

Comment:
can we move this into the java role?

Reply:
That may work but I'm not comfortable with having a role var that depends on a variable that leaves outside of the role (that would go agains t our goal to create independent roles)